Revert "chore: upgrade to Graphql 15 and Graphql-tools 5 and related dependencies"

MounirDhahri opened the pull request:
This PR reverts artsy/metaphysics#5838

**Context:**
We have made two major changes in the past few hours that might lead to conflicts together. The first one being this one https://github.com/artsy/force/pull/14247. That makes it harder to monitor and debug why issues happen.

**Follow-up step**
Merge the PR again once Force is stable and Integrity tests are running fine.


Reverts artsy/metaphysics#5838

damassi commented:
@MounirDhahri - please update the description re why things were reverted (ie, its not a bug, another workstream), then it wont be confusing later if there _is_ an actual bug.

MounirDhahri commented:
Not sure why the build is failing - I can manually see the fields being available

MounirDhahri commented:
Okey I got it - when I did the upgrade, I found show sorts was setting a default value that wasn't available as an enum. I added it as an enum and changed the `defaultValue` from `defaultValue: "-created-at"` to `defaultValue: ShowSorts.getValue(CREATED_AT_DESC)`
